hw/hal/include/hal/hal_system.h

```diff
@@ -66,8 +66,10 @@ enum hal_reset_reason {
     HAL_RESET_SOFT = 4,
     /** Low supply voltage */
     HAL_RESET_BROWNOUT = 5,
+    /** Power on */
+    HAL_RESET_POWER_OFF = 6,
     /** Restart due to user request */
-    HAL_RESET_REQUESTED = 6,
+    HAL_RESET_REQUESTED = 7,
```

Comment:
any reason to change HAL_RESET_REQUESTED value? why not just add POWER_OFF at the end of enum?

Reply:
Good point. Moving it back to original lineup.